docker/autoware-universe/Dockerfile

```diff
@@ -48,6 +48,11 @@ RUN chmod 644 /etc/vulkan/icd.d/nvidia_icd.json
 ADD "https://gitlab.com/nvidia/container-images/opengl/raw/5191cf205d3e4bb1150091f9464499b076104354/glvnd/runtime/10_nvidia.json" /etc/glvnd/egl_vendor.d/10_nvidia.json
 RUN chmod 644 /etc/glvnd/egl_vendor.d/10_nvidia.json
 
+## Regiser OpenCL GPU vendors
```

Comment:
```suggestion
## Register OpenCL GPU vendors
```

Reply:
@angry-crab Reverted?

Reply:
Last commit was not signed and `DCO` check complained :(

Reply:
Regarding DCO, we can make it pass manually, so don't need to care about it too much.